src/utils.js

```diff
@@ -68,14 +68,19 @@ export const pipe = (...fns) => fns.reduce(pipeFns);
 // Document Helpers
 // ==============================
 
+/* calculate scrollbar width to set right padding (prevent view jank) */
 export function getPadding() {
   if (!canUseDOM) return 0;
 
-  const currentPadding = parseInt(document.body.paddingRight, 10) || 0;
-  const clientWidth = document.body ? document.body.clientWidth : 0;
-  const adjustedPadding = window.innerWidth - clientWidth + currentPadding || 0;
+  const bodyMarginLeft = parseInt(window.getComputedStyle(document.body).marginLeft, 10) || 0;
+  const bodyMarginRight = parseInt(window.getComputedStyle(document.body).marginRight, 10) || 0;
+  const bodyMargin = bodyMarginLeft + bodyMarginRight;
+  /* document.body.clientWidth = body width with padding and without margin */
+  const bodyWidth = document.body ? document.body.clientWidth : 0;
```

Comment:
Because of the `canUseDOM` condition on [line 73](https://github.com/jossmac/react-scrolllock/pull/69/files#diff-2b4ca49d4bb0a774c4d4c1672d7aa781R73) we can get rid of the check for `document.body`.

Reply:
Oh yeah. And now I see that we don't have to invoke `window.getComputedStyle(document.body)` two times in two different lines. I'll move that reference into a variable.

Reply:
Redundant. Correct solution outlined in [comment](https://github.com/jossmac/react-scrolllock/pull/69#pullrequestreview-360869745).

Reply:
Changes force pushed. Now we calculate scroll bar width in a simpler way and also take existing (if any) right padding in the body into account.